Extensions/internet connectivity.json

```diff
@@ -0,0 +1,40 @@
+{
+  "author": "The Gem Dev",
+  "description": "",
+  "extensionNamespace": "",
+  "fullName": "Internet connectivity",
+  "name": "Internet_Connectivity",
+  "shortDescription": "an extension which checks if the device is connected to the internet. ",
+  "tags": "javascript, desktop, mobile, internet, connection,",
+  "version": "0.0.1",
+  "eventsFunctions": [
+    {
+      "description": "checks if the device is connected to the internet, when inverted it's set to not connected to the internet.",
```

Comment:
Checks if the device is connected to the internet.

(the rest is self describing)